src/plots/cartesian/index.js

```diff
@@ -198,6 +206,12 @@ exports.drawFramework = function(gd) {
     });
 };
 
+exports.rangePlot = function(gd, plotinfo, cdSubplot) {
+    makeSubplotLayer(plotinfo);
+    plotOne(gd, plotinfo, cdSubplot);
+    Plots.style(gd);
```

Comment:
Yep, I'm not a fan of `Plots.style` too.

As soon as we'll find a solution for removing this ugly block

```js
if(plotinfo.plot) {
  plotinfo.plot.selectAll('g:not(.scatterlayer)').selectAll('g.trace').remove();
}
```

in the main cartesian plot routine, we'll get rid of `Plots.style`